vim-engine/src/main/kotlin/com/maddyhome/idea/vim/vimscript/services/VimVariableServiceBase.kt

```diff
@@ -179,6 +179,7 @@ abstract class VimVariableServiceBase : VariableService {
       "count" -> VimInt(KeyHandler.getInstance().keyHandlerState.commandBuilder.calculateCount0Snapshot())
       "count1" -> VimInt(KeyHandler.getInstance().keyHandlerState.commandBuilder.calculateCount0Snapshot().coerceAtLeast(1))
       "searchforward" -> VimInt(if (injector.searchGroup.getLastSearchDirection() == Direction.FORWARDS) 1 else 0)
+      "hlsearch" -> VimInt(if (injector.searchGroup.isSomeTextHighlighted()) 1 else 0)
```

Comment:
There is a `Boolean.asVimInt(): VimInt` method, but it's also ok

Reply:
Good point, I made the change.